demo/opae-nlb-demo/Dockerfile

```diff
@@ -43,15 +43,15 @@ RUN mkdir /install_root \
 
 # Minimal result image
 FROM scratch as final
-ENV PATH="/usr/local/:${PATH}"
-ENV LD_LIBRARY_PATH="/usr/local/:${LD_LIBRARY_PATH}"
+ENV PATH="/usr/local/bin/:${PATH}"
+ENV LD_LIBRARY_PATH="/usr/local/lib64/:${LD_LIBRARY_PATH}"
```

Comment:
I'd propose to do something like this instead:
```
echo "/usr/local/lib64" | sudo tee -a /etc/ld.so.conf
sudo ldconfig
```

This way /usr/local/lib64 will be used as a standard library path. This would allow to execute binaries that depend on the libraries in /usr/local/lib64 without having to set LD_LIBRARY_PATH.

Reply:
Modified and tested the proposed approach to set /usr/local/lib64 as the standard library path